letsencrypt/rootfs/etc/services.d/lets-encrypt/run

```diff
@@ -166,5 +166,7 @@ fi
 # Get the last modified cert directory and copy the cert and private key to store
 # shellcheck disable=SC2012
 CERT_DIR_LATEST="$(ls -td $CERT_DIR/live/*/ | head -1)"
+mkdir "/ssl"
+chmod 600 "/ssl"
```

Comment:
Hm, that also means that the public key won't  be accessible. Is that really needed? The private key seems to have the proper permissions:

```
-rw-------    1 root     root        1.7K Feb 20 02:00 privkey.pem
```